<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIPatient</class>
 <widget class="QDialog" name="DIPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>病历号：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>病人姓名：</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="2">
         <widget class="QLineEdit" name="leIpname"/>
        </item>
        <item row="13" column="2" colspan="2">
         <widget class="QLineEdit" name="leIpdoa"/>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="leIdwcno"/>
        </item>
        <item row="11" column="2" colspan="2">
         <widget class="QLineEdit" name="leIpag"/>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="leIpmrno"/>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="leIono"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>工作编号：</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2" colspan="2">
         <widget class="QLineEdit" name="leIpsex"/>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>入院日期：</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2" colspan="2">
         <widget class="QLineEdit" name="leIpdod"/>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>病人性别：</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>出院日期：</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>病人年龄：</string>
          </property>
         </widget>
        </item>
        <item row="17" column="3">
         <widget class="QPushButton" name="btnIClose">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>科室编号：</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="2">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="2">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="17" column="1">
         <widget class="QPushButton" name="btnIPatient">
          <property name="text">
           <string>下一个</string>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="16" column="3">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leIpmrno</tabstop>
  <tabstop>leIono</tabstop>
  <tabstop>leIdwcno</tabstop>
  <tabstop>leIpname</tabstop>
  <tabstop>leIpsex</tabstop>
  <tabstop>leIpag</tabstop>
  <tabstop>leIpdoa</tabstop>
  <tabstop>leIpdod</tabstop>
  <tabstop>btnIPatient</tabstop>
  <tabstop>btnIClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
